client/src/partials/profit_center/center/analysis_profit_center.js

```diff
@@ -9,7 +9,8 @@ angular.module('bhima.controllers')
   'SessionService',
```

Comment:
Bruce you can begin to use to use aliasing in the controller to make the code more clear.